manifests/config/apache.pp

```diff
@@ -200,6 +203,28 @@
     include ::apache::mod::intercept_form_submit
     include ::apache::mod::lookup_identity
     include ::apache::mod::auth_kerb
+  } elsif $keycloak {
+    apache::mod { 'auth_openidc':
+      package => 'mod_auth_openidc',
+    }
```

Comment:
Should we also install keycloak-httpd-client-install package too?

Reply:
I was debating that, but decided against it. I think the instruction should be:
* `yum install keycloak-httpd-client-install`
* `keycloak-httpd-client-install ....`
* `foreman-installer --foreman-keycloak true --foreman-keycloak-realm ...`

In theory the installer could run the install command, but then we need to pass credentials and I think that's something we probably don't want.